src/channel.ts

```diff
@@ -1658,7 +1658,7 @@ export class Channel<
     if (!this.state.messages) {
       this.state.messages = Immutable([]);
     }
-    this.state.addMessagesSorted(messages, true);
+    this.state.addMessagesSorted(messages, false, true);
```

Comment:
Having options an object with keys would help for readability.

Reply:
3 params are fine, in case it grows we can change it to object

Reply:
I agree in general but when they are booleans, it's trickier.